src/applications/lgy/coe/form/content/LoggedInContent.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
```

Comment:
Is OMBInfo still being used in other files or can we go ahead and remove it from our component-library?

Reply:
This is the only file that uses `OMBInfo`